Forcepoint/forcepoint-swg/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,34 @@
+[
+    {
+      "value": "Connection from {source.ip} to {url.original} blocked: {event.reason}",
+      "conditions": [
```

Comment:
I recommend you specify in your `conditions` every fields that you use within your smart description.

Otherwise there is some situations where we have partial smart descriptions.